app/src/org/commcare/dalvik/activities/CommCareSetupActivity.java

```diff
@@ -550,22 +556,103 @@ protected void deliverError(CommCareSetupActivity receiver,
     public boolean onCreateOptionsMenu(Menu menu) {
         super.onCreateOptionsMenu(menu);
         menu.add(0, MODE_ARCHIVE, 0, Localization.get("menu.archive")).setIcon(android.R.drawable.ic_menu_upload);
+        menu.add(0, MODE_SMS, 1, Localization.get("menu.sms")).setIcon(android.R.drawable.ic_menu_upload);

```

Comment:
The icon that is set is the same as the `archive` menu entry found above. This is only relevant to Android 2 devices, but it is worth choosing something other than `ic_menu_upload`.

Maybe `stat_notify_chat` 
![image](https://cloud.githubusercontent.com/assets/94817/9858927/7a96305e-5af1-11e5-9f51-5aeaa6d37d06.png)
or something else from http://www.darshancomputing.com/android/1.5-drawables.html